### ISSUE TITLE
Keyboard covers toast

### ISSUE THREAD

**Issue by AlirezaAkbarix** [magicismight/react-native-root-toast#67]
When keyboard is up and toast position is bottom, it's covered by keyboard! Is it possible to come upon keyboard view?

**Comment by prancky** [comment]
+

**Comment by prancky** [comment]
@AlirezaAkbarix  i found a way to fix it. here is my solution.
In button press event, i called  `this._passwordInput.blur();`
"_passwordInput" is my input feild ref.

**Comment by Jinzm** [comment]
maybe the only way to fix it to make keyboard down

**Comment by haven2world** [comment]
I think you can try to show the toast in the middle of the screen. In fact, I have already done this.

**Comment by luisgomesredicom** [comment]
Yes, it is possible to close the phone keyboard in React Native. To do this, you can use React Native's **TouchableWithoutFeedback** component along with the **Keyboard.dismiss()** function to close the keyboard.
